taxonomies/food/categories.txt

```diff
@@ -12832,7 +12832,6 @@ tr: Soğuk kahve
 wikidata:en: Q968554
 
 < en:Non-alcoholic beverages
-< en:Wines
```

Comment:
It _might_ be better to subcategorise Wines in alcoholic and non-alcoholic (and move the `< en:Alcoholic beverages` to the former group), but I also get that that’s a larger change, and for now it’s better to not have Non-alcoholic wines be children of Alcoholic beverages.